src/SmartTransactionsController.ts

```diff
@@ -42,23 +44,27 @@ export const DEFAULT_INTERVAL = SECOND * 5;
 export type SmartTransactionsControllerConfig = BaseConfig & {
   interval: number;
   clientId: string;
-  chainId: string;
+  chainId: Hex;
   supportedChainIds: string[];
 };
 
+type FeeEstimates = {
+  approvalTxFees: IndividualTxFees | undefined;
+  tradeTxFees: IndividualTxFees | undefined;
+};
+
 export type SmartTransactionsControllerState = BaseState & {
   smartTransactionsState: {
-    smartTransactions: Record<string, SmartTransaction[]>;
+    smartTransactions: Record<Hex, SmartTransaction[]>;
     userOptIn: boolean | undefined;
     liveness: boolean | undefined;
-    fees: {
-      approvalTxFees: IndividualTxFees | undefined;
-      tradeTxFees: IndividualTxFees | undefined;
-    };
+    fees: FeeEstimates;
+    feesByChainId: Record<Hex, FeeEstimates>;
+    livenessByChainId: Record<Hex, boolean | undefined>;
```

Comment:
```suggestion
    livenessByChainId: Record<Hex, boolean>;
```
seems like we can always ensure that we set a bool?